### DISCUSSION TITLE
【讨论】大家怎么看用脚本代绳网接委托？

### DISCUSSION THREAD

**Discussion by FFT66** [share121/inter-knot#1450] [其他]
最近看到很多人用脚本接委托，不知道大家怎么看待这件事情

**Comment by dengyunzhe** [comment]
这个委托我接了

**Comment by lllliufeng** [comment]
这不抢了我们人工队的工作吗？强烈抗议！！！

**Comment by ran145s** [comment]
我上有老下有小，就靠接个委托混口饭吃
这些脚本真是害人不浅啊！！！

**Comment by Xieleng42** [comment]
现在接委托真的是越来越难了，唉……

**Comment by ZMJ2333** [comment]
我的评价是……找超高难度的委托接就是了

**Comment by qiuledadada** [comment]
这年头生意不好做了啊……可恨的脚本！

**Comment by dudu-ke** [comment]
不会被封吗(=_=)

**Comment by GanlinSama** [comment]
话说回来这些个开脚本的估计也不好受，一觉醒来手头上多了一堆杂七杂八的委托，有的可能自己还处理不了，这不头大

**Comment by Inter-Knot-User** [comment]
建议绳网查询一下是否用户使用脚本，如果有使用，直接屏蔽ip

**Comment by GanlinSama** [comment]
woc，布什各门，赛博生命这么快就打过来了？？？
